pkg/server/router/dwn.go

```diff
@@ -0,0 +1,103 @@
+package router
+
+import (
+	"context"
+	"fmt"
+	"github.com/TBD54566975/ssi-sdk/credential/manifest"
+	"github.com/tbd54566975/ssi-service/pkg/service/dwn"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/tbd54566975/ssi-service/pkg/server/framework"
+	svcframework "github.com/tbd54566975/ssi-service/pkg/service/framework"
+)
+
+type DWNRouter struct {
+	service *dwn.Service
+}
+
+func NewDWNRouter(s svcframework.Service) (*DWNRouter, error) {
+	if s == nil {
+		return nil, errors.New("service cannot be nil")
+	}
+	dwnService, ok := s.(*dwn.Service)
+	if !ok {
+		return nil, fmt.Errorf("could not create dwn router with service type: %s", s.Type())
+	}
+
+	return &DWNRouter{
+		service: dwnService,
+	}, nil
+}
+
+type DWNPostRequest struct {
+	Manifest manifest.CredentialManifest `json:"manifest" validate:"required"`
+}
+
+type PublishManifestRequest struct {
```

Comment:
I'm thinking this kind of request should go in a different package

maybe there should be an `dwn` package that has `requests.go` with this kind of stuff?